File: JSfundamentals/josiah_algo.py

p = ['butt', 'm', 'x', 'o', 'p']
y = 'x'

def change_x(arr, x):
    for i in range(0, len(arr)):
        if arr[i] == y:
            arr[i] = "we changed X but we cant pop"
    return arr
print change_x(p,y)

File: JSfundamentals/josiah_algo.js
var arr_test = ['butt', 'm', 'x', 'o', 'p'];
var x_test = 'x';
function remove_x(arr, x) {
for (var i = 0; i < arr.length -1; i++) {
    console.log(arr[i] + "**********")
    if(arr[i] == x) {
        arr.slice(arr[i])
    }
    return arr
}

}

console.log(remove_x(arr_test, x_test))
